spec/integration/alerts_spec.rb

```diff
@@ -623,6 +655,17 @@ module Hawkular::Alerts::RSpec
       expect(events.size).to be >= 12
     end
 
+    it 'Should list events for multiple tenants' do
+      client = Hawkular::Alerts::Client.new(ALERTS_BASE, creds, options)
+
+      events = client.events(criteria: { 'thin' => true }, tenants: [:hawkular, :foo])
+
+      expect(events).to_not be_nil
+      expect(events.size).to be >= 24
+      expect(events.count { |e| e.tenantId == 'hawkular' }).to be >= 12
+      expect(events.count { |e| e.tenantId == 'foo' }).to be >= 12
```

Comment:
That's possible. 
Is also possible that someone running the tests already had some events on their system. 
Recording new cassettes under those conditions is annoying...  that's why i left the >= in other parts of the code.